lib/galaxy/dependencies/conditional-requirements.txt

```diff
@@ -1,5 +1,5 @@
 # These dependencies are only required when certain config options are set
-psycopg2==2.7.3
+psycopg2-binary==2.7.4
```

Comment:
If we change the name of this - it won't upgrade older versions of this when upgrading Galaxy right? This could still be the right thing to do but maybe we need to think about how to handle that - should we uninstall psycopg2 in common startup?

Reply:
@jmchilton Good point, I can check what happens but I don't know how this would behave off the top of my head if you have both installed.

Reply:
To update this comment, which was subsequently discussed on IRC - the suspicion was correct.  Installing psycopg2-binary does *not* automatically uninstall psycopg2 even though they share the same namespace in site-packages.  This leaves you in a weird situation where pip thinks both are installed, but really only the most recently installed library is intact.

Uninstalling the previous lib will mangle the new one, and so taking care to manually remove psycopg2==2.4.3 and removing it prior to installing a new psycopg2-binary version is what I've implemented in subsequent commits.